src/client.js

```diff
@@ -23,12 +23,29 @@ const API_TYPE_CLIENT = 'client';
 const API_TYPE_APP = 'app';
 
 /**
- * Create a new StitchClient instance.
+  * Factory class to create a new StitchClient asynchronously.
+  */
+export class StitchClient {
```

Comment:
[minor] Same comment about having just one combined class that we export as we discussed earlier.